_rules/SC1-2-1-audio-transcript.md

```diff
@@ -21,26 +18,28 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or with a "play button".
+The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or has a "play button".  If a play button exists the rule applies if the button is [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
 
-A play button is an interactive element that when activated, plays the audio. The play button must be [visible on the page](#visible-on-the-page) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+A play button is an interactive element that when activated, plays the audio. The play button must be [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
 
 ### Expectation
 
-The auditory information of each test target is available through a caption, or through a [text transcript](#text-transcript) that is available either on the page or through a link.
+The auditory information of each test target is available through a text transcript that is available either on the page or through a link.
```

Comment:
Do we want to further define "available"? For example, "presented to the user" vs "included in the accessibility tree" etc? Just a thought.